gdk4/src/cairo_interaction.rs

```diff
@@ -45,7 +45,7 @@ pub trait GdkCairoContextExt {
     );
 
     #[doc(alias = "gdk_cairo_set_source_rgba")]
```

Comment:
Please add a doc alias for set_source_rgba

Reply:
Also we should probably move the functions implementations to the trait definition and seal it. Would you mind doing that in a separate commit here? :)

Reply:
Sure